guides/non-tee.md

```diff
@@ -0,0 +1,245 @@
+# Running a workload
+
+## Creating a sample CoCo workload
+
+Once you've used the operator to install Confidential Containers, you can run a pod with CoCo by simply adding a runtime class.
+First, we will use the `kata` runtime class which uses CoCo without hardware support.
```

Comment:
I thought this was changed to `kata-qemu-coco-dev`
@fidencio @stevenhorsman ?

Reply:
Yes - `kata-qemu-coco-dev` is correct for the non-TEE hardware CoCo support for dev/test

Reply:
Updated with the right runtimeclass name. @stevenhorsman @bpradipt could you please review the file one more time, to make sure, that it looks good and nothing else is missing?